### DISCUSSION TITLE
Configuring Mumble-Server/I2pd Tunnel

### DISCUSSION THREAD

**Discussion by I2pRandom** [PurpleI2P/i2pd#1866] [Q&A]
So i've managed to finally get a tunnel working to get a connection through to my Mumble server. 
But i cannot seem to get TCP connections through, is there something i'm missing? i've omitted the keys.

```
[MATTER-TCP]
type = server
host = 127.0.0.1
port = 8086

[MATTER-UDP]
type = udpserver
host = 127.0.0.1
port = 8086
```

**Comment by r4sas** [comment]
```ini
[MUMBLE]
type = server
host = 127.0.0.1
port = 64738
gzip = false
inbound.length = 1
outbound.length = 1
inbound.quantity = 5
outbound.quantity = 5
keys = mumble.dat

[MUMBLE-UDP]
type = udpserver
host = 127.0.0.1
port = 64738
keys = mumble.dat
```

You need both tunnels to use same key. Second one tunnel uses same i2cp options as first (because they use one key), so they can be skipped.

**Reply by I2pRandom** [reply]
TCP pings are still failing to get through, the server fails to respond back. 
it's strange, idk why it's failing because they do both use the same key.
